frontend/src/services/enquiryService.ts

```diff
@@ -33,6 +33,14 @@ export default {
     return appAxios().get(`enquiry/${enquiryId}`);
   },
 
+  /**
+   * @function listRelatedEnquiries
+   * @returns {Promise} An axios response
```

Comment:
JSDoc: This should have a description as well as an `@param` field describing the `activityId`